server/src/main/kotlin/org/jetbrains/bsp/bazel/server/sync/ExecuteService.kt

```diff
@@ -128,6 +128,9 @@ class ExecuteService(
         return CleanCacheResult(true)
     }
 
+    // TODO handle properly
```

Comment:
pls create a YT ticket

Reply:
The function are now handled properly

Reply:
soo is the todo needed?

Reply:
Oh sorry, I forgot to delete that before